build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -exuo pipefail
+
+yarn workspace @abacus-network/utils run build
```

Comment:
definitely would prefer to
1. avoid adding additional root level files
2. use the same local and CI invocation
adding a package and forgetting to include it here in eg lint is pretty annoying